config/file_test.go

```diff
@@ -56,7 +57,8 @@ func getActualFileConfig(t *testing.T, path string) *model.Config {
 	require.Nil(t, err)
 	defer f.Close()
 
-	actualCfg, _, err := config.UnmarshalConfig(f, false)
+	var actualCfg *model.Config
+	err = json.NewDecoder(f).Decode(&actualCfg)
```

Comment:
Any reason for not doing "json.Unmarshal"?

Reply:
Decoder is less lines.